cinema/urls.py

```diff
@@ -0,0 +1,17 @@
+from django.urls import path
+from cinema.views import MovieListCreateView, MovieRetrieveUpdateDestroyView
+
+urlpatterns = [
+    path(
+        "api/cinema/movies/",
+        MovieListCreateView.as_view(),
+        name="movie-list-create"
+    ),
+    path(
+        "api/cinema/movies/<int:pk>/",
```

Comment:
u can add similar part to a urls.py in the main folder of a project before include